modes/segmentation/src/index.tsx

```diff
@@ -132,7 +132,9 @@ function modeFactory({ modeConfiguration }) {
             id: ohif.layout,
             props: {
               leftPanels: [ohif.leftPanel],
+              leftPanelResizable: true,
```

Comment:
Do we want to make the default resizable? If so consider changing the corresponding ViewerLayout props to have a default value of true.

Reply:
It's up to Alireza, I think the default should be true and you can just set it to false here, so we don't run into confusing scenarios like Bill did. @sedghi

Reply:
Let's make it resizable, these are our mode, we can do whatever we want. We should also apply similar changes to longitudinal and all other modes, thanks